src/ipc-handlers.ts

```diff
@@ -909,12 +909,24 @@ export async function showErrorMessageBox(
 		/Error invoking remote method '\w+': Error:/g,
 		''
 	);
-	await showMessageBox( event, {
+
+	console.error( filteredError );
+
+	const response = await showMessageBox( event, {
 		type: 'error',
 		message: title,
-		detail: error ? `${ message }\n\nError: ${ filteredError }` : message,
-		buttons: [ __( 'OK' ) ],
+		detail: message,
+		buttons: [ __( 'Open Logs' ), __( 'OK' ) ],
```

Comment:
Could we make it clearer on what logs it opens:

```suggestion
		buttons: [ __( 'Open Studio logs' ), __( 'OK' ) ],
```